app/js/components/LabResultsList.jsx

```diff
@@ -127,6 +131,31 @@ export class LabResultsList extends PureComponent {
     }
   }
 
+  filterData(filters, data, locale) {
```

Comment:
this is a modified version of the old filterThrough method, targeted specifically for this page.